src/client/components/PaginationPanel/i18n/de.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  PaginationPanel: {
+    goToPage: 'Gehe zu Seite'
```

Comment:
Translation is too long and means more than just 'Go', we would not have so much space for a button.